dash/dash.py

```diff
@@ -1434,14 +1429,22 @@ def _validate_layout(self):
             component_ids.add(component_id)
 
     def _setup_server(self):
+        # Apply _force_eager_loading overrides from modules
+        eager_loading = self.config.eager_loading
+        for module_name in ComponentRegistry.registry:
+            module = sys.modules[module_name]
+            eager = getattr(module, '_force_eager_loading', False)
+            eager_loading = eager_loading or eager
+
+        # static files from the packages
+        self.css = Css(self.css.config.serve_locally)
+        self.scripts = Scripts(self.scripts.config.serve_locally, eager_loading)
+
         if self.config.include_assets_files:
             self._walk_assets_directory()
 
         self._validate_layout()
 
-        self._generate_scripts_html()
-        self._generate_css_dist_html()
```

Comment:
Seems like we need these for their side-effects, or perhaps recreating `self.scripts` and `self.css` was the problem? Regardless, I don't see any harm bringing this back.

Reply:
Agreed. Undoing this.